docs/man_pages/project/testing/appstore-upload.md

```diff
@@ -0,0 +1,52 @@
+appstore upload
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns appstore upload [<Apple ID> [<Password> [<Mobile Provisioning Profile Identifier> [<Code Sign Identity>]]]]]`
+General | `$ tns appstore upload [<Apple ID> [<Password>]] --ipa <Ipa File Path>`
+
+Uploads project to iTunes Connect.
```

Comment:
Make sure to explain that users can issue a production build which will build the project and upload it OR they can pick an already built package and upload it.